Stage

LalithaBanda opened the pull request:
Developed fallowing Tests,
ChangeCC_HeaderLevel
ChangeQuantity_ExistingLineLevelRequisition
Verify_AddInternalComments
Verify_ChangeRequireBy_ExistingLineLevel
Verify_ChangeRequireBy_HeaderLevel
Verify_CopiedItem_LineLevel 
Verify_ItemDelete
Verify_SaveCart

LalithaBanda committed:
Developed new Methods and updated Timeouts

LalithaBanda committed:
Developed new Test "ChangeCC_HeaderLevel"
Added new WorkBook "UpdateCostCenter"
Changed Existing Test with Workbook filename and Sheetname

PraveenNamburi committed:
Added TestCase in sandbox which is under development -
VerifyRequester_changeRequire_ExistingLineLevel.

PraveenNamburi committed:
Merge branch 'lalitha' of https://github.com/Orasi/Xeeva.git into
lalitha

Conflicts:
	src/main/java/com/xeeva/catalog/RequisitioningPage.java
	src/main/java/com/xeeva/navigation/MainNav.java
	src/test/java/catalog/cartCheckoutProcess/ChangeCC_LineLevel.java
	src/test/java/catalog/nonPriceAgreementItems/AddNonPriceAgreementItem_FromRecentOrders.java
	src/test/java/catalog/priceAgreementItems/Add_PriceAgreementItem_FavouriteFolder.java

PraveenNamburi committed:
Developed a new test in catalog module -
VerifyRequestor_changeRequire_ExistingLineLevel.

LalithaBanda committed:
Updated Existing WorkBook with new input

LalithaBanda committed:
Added new Methods

LalithaBanda committed:
Developed new Test "ChangeQuantity_ExistingLineLevelRequisition"

LalithaBanda committed:
Developed new Test "Verify_CopiedItem_LineLevel"

LalithaBanda committed:
Merge branch 'lalitha' of https://github.com/Orasi/Xeeva.git into
lalitha

Conflicts:
	src/main/java/com/xeeva/catalog/CostCenterPage.java
	src/test/resources/datasheets/UpdateCostCenter.xlsx

PraveenNamburi committed:
Developed a new test in catalog module -
VerifyRequester_changeRequire_ExistingLineLevel.

PraveenNamburi committed:
1. Developed a new test in catalog module -
Verify_ChangeRequireBy_HeaderLevel.
2. Updated the test class -
VerifyRequester_changeRequire_ExistingLineLevel.
3. Added the following methods in CostCenterPage:
   i. change_RequiredByAtLineLevel, and
   ii.change_RequiredByAtHeaderLevel.

PraveenNamburi committed:
Adding file to index.

PraveenNamburi committed:
Renamed the test as - Verify_ChangeRequireBy_ExistingLineLevel.

LalithaBanda committed:
Developed new Test "Verify_SaveCart"

LalithaBanda committed:
Merge branch 'lalitha' of https://github.com/Orasi/Xeeva.git into
lalitha

Conflicts:
	src/main/java/com/xeeva/catalog/CostCenterPage.java

PraveenNamburi committed:
1.Developed a new test in catalog module -
CancelRequisiton_RecentOrdersList.
2. Added a new method in Requisitioning page -
cancelRequisitionFromRecentOrders(String comments).

PraveenNamburi committed:
Merge branch 'lalitha' of https://github.com/Orasi/Xeeva.git into lalitha

LalithaBanda committed:
Added new Methods , Updated Sync Timeouts

LalithaBanda committed:
Developed new Test "Verify_ItemDelete"

LalithaBanda committed:
Merge branch 'lalitha' of https://github.com/Orasi/Xeeva.git into
lalitha

Conflicts:
	src/main/java/com/xeeva/catalog/RequisitioningPage.java

LalithaBanda committed:
Merged

LalithaBanda committed:
Merge branch 'lalitha' of https://github.com/Orasi/Xeeva into stage

# Conflicts:
#	src/main/java/com/xeeva/catalog/RequisitioningPage.java
#	src/main/java/com/xeeva/navigation/MainNav.java
#	src/test/java/catalog/cartCheckoutProcess/ChangeCC_LineLevel.java
#	src/test/java/catalog/nonPriceAgreementItems/AddNonPriceAgreementItem_FromFavoriteItem.java
#	src/test/java/catalog/nonPriceAgreementItems/AddNonPriceAgreementItem_FromRecentOrders.java
#	src/test/java/catalog/priceAgreementItems/Add_PriceAgreementItem_FavouriteFolder.java

jessicamarshall commented on src/main/java/com/xeeva/catalog/CostCenterPage.java:
```diff
@@ -112,30 +135,122 @@ public void AddInternalComment(String InternalComment){
+				AlertHandler.handleAlert(driver, 6);
+			}
+			// Verify Delete 
+			Sleeper.sleep(9000);
```
This is a long hard wait, is there another way we can wait for the delete to finish?

jessicamarshall commented on src/main/java/com/xeeva/catalog/CostCenterPage.java:
```diff
@@ -152,16 +267,172 @@ public void change_CC(String changeType,String CC){
 		}
 	}
-}
 
```
this method needs a header

jessicamarshall commented on src/main/java/com/xeeva/catalog/CostCenterPage.java:
```diff
@@ -152,16 +267,172 @@ public void change_CC(String changeType,String CC){
-}
 
+	public String selectCCToCopyItem(){
+		driver.setPageTimeout(4);
```
There is no need to change the Page timeout here.  All the page time out does is change the time it takes for a page to load when using the commands driver.get(URL) or driver.navigate(URL).  Please remove

jessicamarshall commented on src/main/java/com/xeeva/catalog/CostCenterPage.java:
```diff
@@ -152,16 +267,172 @@ public void change_CC(String changeType,String CC){
+		driver.setPageTimeout(4);
+		String ccValue = lstSelectCCValue.get(0).getText();
+		System.out.println("CostCenter Name "+lstSelectCCValue.get(0).getText());
+		Sleeper.sleep(5000);
```
Why is there a 5 second wait between getting the text from an element and then clicking an element?  Seems unnecessary.  Please remove.

jessicamarshall commented on src/main/java/com/xeeva/catalog/CostCenterPage.java:
```diff
@@ -152,16 +267,172 @@ public void change_CC(String changeType,String CC){
+		// after cart check out, application taking time to load cost center page 
+		lblCC.syncVisible(30, false);
+		lblCC.isDisplayed();
+		driver.setPageTimeout(3);
```
No need to change the page timeout here...again, changing page timeout only changes the time a page takes to load when using the commands driver.get(URL) or driver.navigate(URL).  PLease remove.

jessicamarshall commented on src/main/java/com/xeeva/catalog/CostCenterPage.java:
```diff
@@ -152,16 +267,172 @@ public void change_CC(String changeType,String CC){
+		int getNextMonthArrowsSize = nextMonthArrows.size();
+		System.out.println("Next month arrows size: "+getNextMonthArrowsSize);
+		for(WebElement nextMonth : nextMonthArrows){
+			driver.setPageTimeout(2);
```
Please remove

jessicamarshall commented on src/main/java/com/xeeva/catalog/CostCenterPage.java:
```diff
@@ -152,16 +267,172 @@ public void change_CC(String changeType,String CC){
+		for(WebElement nextMonth : nextMonthArrows){
+			driver.setPageTimeout(2);
+			nextMonth.click();
+			driver.setPageTimeout(2);
```
Please remove

jessicamarshall commented on src/main/java/com/xeeva/catalog/CostCenterPage.java:
```diff
@@ -152,16 +267,172 @@ public void change_CC(String changeType,String CC){
+
+		//Verify the message -'The RequiredBy date has been updated' is displayed.
+		verify_RequiredByDateUpdated_atCCLevel();
+		driver.setPageTimeout(2);
```
Please remove

jessicamarshall commented on src/main/java/com/xeeva/catalog/CostCenterPage.java:
```diff
@@ -152,16 +267,172 @@ public void change_CC(String changeType,String CC){
+		// after cart check out, application taking time to load cost center page 
+		lblCC.syncVisible(30, false);
+		lblCC.isDisplayed();
+		driver.setPageTimeout(3);
```
Please remove the setPageTimeOut method.  If you're trying to change the timeout when looking for an element - you would want to use driver.setElementTimeout(3).  But then after you're done, you need to return to the default timeout driver.setElementTimeout(Constants.ELEMENT_TIMEOUT);

jessicamarshall commented on src/main/java/com/xeeva/catalog/CostCenterPage.java:
```diff
@@ -152,16 +267,172 @@ public void change_CC(String changeType,String CC){
+				+ "table/thead/tr[2]/td[4]"));
+		int loopCount=0;
+		for(WebElement nextMonth : nextMonthArrows){
+			driver.setPageTimeout(2);
```
Here you need to just remove the driver.setPageTimeout(2).  You can't really use an the setElementTimeout as it isn't applicable - only works when using a driver.findElementBy() or driver.findElementsBy() and as you are using the page factory element you've defined (nextMonth) that is already stored in memory and doesn't use the findElement method.

jessicamarshall commented on src/main/java/com/xeeva/catalog/CostCenterPage.java:
```diff
@@ -152,16 +267,172 @@ public void change_CC(String changeType,String CC){
+		for(WebElement nextMonth : nextMonthArrows){
+			driver.setPageTimeout(2);
+			nextMonth.click();
+			driver.setPageTimeout(2);
```
Please remove

jessicamarshall commented on src/main/java/com/xeeva/catalog/ItemDetailsPage.java:
```diff
@@ -140,13 +147,22 @@ public void modifyItemDetails(String UnitPrice, String Quantity, String strUOMVa
 	 **/
 	public void add_TwoDiffrent_ItemsToCart(String UP,String Qty,String UOM){
```
This method needs a header please

jessicamarshall commented on src/main/java/com/xeeva/catalog/ItemDetailsPage.java:
```diff
@@ -140,13 +147,22 @@ public void modifyItemDetails(String UnitPrice, String Quantity, String strUOMVa
 		TestReporter.logStep("UOM "+lstUOMGlobalCart.size());
-		TestReporter.logStep("Add To Cart buttons "+btnIAddToCartItems.size());
+		TestReporter.logStep("Add To Cart Items : "+btnIAddToCartItems.size());
+		driver.setPageTimeout(10, TimeUnit.SECONDS);
```
Please remove

jessicamarshall commented on src/main/java/com/xeeva/catalog/ItemDetailsPage.java:
```diff
@@ -140,13 +147,22 @@ public void modifyItemDetails(String UnitPrice, String Quantity, String strUOMVa
+		driver.setPageTimeout(5, TimeUnit.SECONDS);
 		lstUOMGlobalCart.get(1).select(UOM);
 		driver.executeJavaScript("arguments[0].click();", btnIAddToCartItems.get(1));
-		driver.manage().timeouts().implicitlyWait(5, TimeUnit.SECONDS);
```
Please remove - again - the implicitlyWait (same as driver.setElementTimeout()), only changes timeout if you are using driver.findElement methods

jessicamarshall commented on src/main/java/com/xeeva/catalog/ItemDetailsPage.java:
```diff
@@ -140,13 +147,22 @@ public void modifyItemDetails(String UnitPrice, String Quantity, String strUOMVa
+		  if(AlertHandler.isAlertPresent(driver, 6)){
+		   AlertHandler.handleAlert(driver, 6);
+		  }
+		driver.setPageTimeout(5, TimeUnit.SECONDS);
```
Please remove

jessicamarshall commented on src/main/java/com/xeeva/catalog/ItemDetailsPage.java:
```diff
@@ -140,13 +147,22 @@ public void modifyItemDetails(String UnitPrice, String Quantity, String strUOMVa
+		  if(AlertHandler.isAlertPresent(driver, 6)){
+		   AlertHandler.handleAlert(driver, 6);
+		  }
+		driver.setPageTimeout(5, TimeUnit.SECONDS);
```
Please remove

jessicamarshall commented on src/main/java/com/xeeva/catalog/RequisitioningPage.java:
```diff
@@ -111,9 +118,9 @@ private void pageLoaded(){
+		ReqTab.syncVisible(20, false);
 		ReqTab.click();
-		driver.setPageTimeout(2);
+		driver.setPageTimeout(5);
```
Please remove

jessicamarshall commented on src/main/java/com/xeeva/catalog/RequisitioningPage.java:
```diff
@@ -225,7 +232,7 @@ public void  perform_CatalogSearch(String searchItem){
 		searchButton.syncVisible(20,false);
 		searchButton.click();
-		driver.setPageTimeout(4);
+		driver.manage().timeouts().implicitlyWait(20, TimeUnit.SECONDS);
```
Please remove

jessicamarshall commented on src/main/java/com/xeeva/catalog/RequisitioningPage.java:
```diff
@@ -421,38 +428,78 @@ public void click_FavoriteItem(){
+		  btnSeeMore.syncEnabled(5);
+		  //btnSeeMore.click();
+		  driver.executeJavaScript("arguments[0].click();", btnSeeMore);
+		  driver.setPageTimeout(5);
```
Please remove

jessicamarshall commented on src/main/java/com/xeeva/catalog/RequisitioningPage.java:
```diff
@@ -421,38 +428,78 @@ public void click_FavoriteItem(){
+				String getStatus = driver.findElement(By.xpath("//*[@id='gvRecentOdersGrid']/tbody/"
+						+ "tr["+ row +"]/td[8]/span")).getText();
+				if(!getStatus.contains("Canceled By Requester")){
+					driver.setPageTimeout(2);
```
Please remove.  If you're wanting to change the element time out for the next statement, you can use driver.SetElementTimeout(2) but after you do the next line, need to return to the default driver.setElementTimeout(Constants.ELEMENT_TIMEOUT);

jessicamarshall commented on src/main/java/com/xeeva/catalog/SearchItems/GlobalItemsTab.java:
```diff
@@ -53,11 +53,9 @@ private void pageLoaded(){
 		driver.executeJavaScript("arguments[0].click();", globalItemsTab);
-		//globalItemsTab.click();
-		driver.setPageTimeout(3);
+		driver.setPageTimeout(5);
```
Please remove

jessicamarshall commented on src/main/java/com/xeeva/navigation/MainNav.java:
```diff
@@ -255,8 +255,7 @@ public void perform_CartItemVerifications(String UnitPrice,String UOM,String Qua
-		// lblCartValue element is not finding with page,element,script timeouts
-		Sleeper.sleep(10000);
+		Sleeper.sleep(15000);
 		TestReporter.logStep("Cart Having [" + lblCartValue.getText()+"] Items!!");
```
15 seconds is long time to hard code a wait.  Can we revisit this and find a way to do a dynamic wait?  Let me know if you want me to take a look and help

Commit:
worked on some sync issues with the cart pop up and the global item page tab

LalithaBanda committed:
Fixed all changes as per review comments

PraveenNamburi committed:
1. Developed a new test in catalog module -
VerifyReq_SearchRejectedOrders.
2. Added the following methods in Recent Order Information page:
  a.getCart_RejectedOrders();
  b.getRFQ_RejectedOrders();
  c.getREQ_RejectedOrders(); and
  d.enterSearchCriteriaAndVerifyRejectedOrders().

PraveenNamburi committed:
Merge branch 'lalitha' of https://github.com/Orasi/Xeeva.git into
lalitha

Conflicts:
	src/main/java/com/xeeva/catalog/RequisitioningPage.java

PraveenNamburi committed:
Removed all the driver.setPageTimeOut(); statements.

PraveenNamburi committed:
Uncommented the line - @AfterMethod() in the test -
Verify_ChangeRequireBy_HeaderLevel.

jessicamarshall commented on src/main/java/com/xeeva/catalog/CostCenterPage.java:
```diff
@@ -1,438 +1,448 @@
+	 */
+	public void AddInternalComment(String InternalComment){
+		lblCC.syncVisible(30, false);
+		lblCC.isDisplayed();
```
this method isDisplayed() - only returns a true or false if the element is displayed.  It doesn't wait for an element to display.  So if you are not doing anything with the return value, then you need to remove this statment.

jessicamarshall commented on src/main/java/com/xeeva/catalog/CostCenterPage.java:
```diff
@@ -1,438 +1,448 @@
+		}else{
+			TestReporter.assertTrue(false, "Internal Comment Wrongly Displayed");
+		}
+		driver.setPageTimeout(2);
```
Please remove this pageTimeout(2)

jessicamarshall commented on src/main/java/com/xeeva/catalog/CostCenterPage.java:
```diff
@@ -1,438 +1,448 @@
+		lblCC.syncVisible(30, false);
+		lblCC.isDisplayed();
+		driver.setElementTimeout(Constants.ELEMENT_TIMEOUT);
+
```
Need to remove line 358, if it is not returning any value and doing something with it.   ALso need to remove the setElementTimeout, as you are setting it to the default - which it should already be at.

jessicamarshall commented on src/main/java/com/xeeva/catalog/CostCenterPage.java:
```diff
@@ -1,438 +1,448 @@
+		lblCC.syncVisible(30, false);
+		lblCC.isDisplayed();
+		driver.setPageTimeout(3);
+		//To click on calender at CC_HeaderLevel.
```
Again - need to remove lines 405 and 406

jessicamarshall replied:
If you are wanting to change the element timeout for the List element you are doing below (line 409), you can change the element timeout to whatever you, want but then after, you'll need to set back to the default value.

jessicamarshall commented on src/main/java/com/xeeva/catalog/CostCenterPage.java:
```diff
@@ -1,438 +1,448 @@
+				+ "table/thead/tr[2]/td[4]"));
+		int loopCount=0;
+		for(WebElement nextMonth : nextMonthArrows){
+			driver.setPageTimeout(2);
```
Remove please

jessicamarshall commented on src/main/java/com/xeeva/catalog/CostCenterPage.java:
```diff
@@ -1,438 +1,448 @@
+		for(WebElement nextMonth : nextMonthArrows){
+			driver.setPageTimeout(2);
+			nextMonth.click();
+			driver.setPageTimeout(2);
```
Remove please

jessicamarshall commented on src/main/java/com/xeeva/catalog/CostCenterPage.java:
```diff
@@ -1,438 +1,448 @@
+					+ "table/tbody/tr[4]/td[@class='day']"));
+			for(WebElement selWeekDate : selectDates){
+				selWeekDate.click();
+				driver.setPageTimeout(5);
```
Remove please

jessicamarshall commented on src/main/java/com/xeeva/catalog/CostCenterPage.java:
```diff
@@ -1,438 +1,448 @@
+
+		//Verify the message -'The RequiredBy date has been updated' is displayed.
+		verify_RequiredByDateUpdated_atCCLevel();
+		driver.setPageTimeout(2);
```
Remove please

jessicamarshall commented on src/main/java/com/xeeva/catalog/ItemDetailsPage.java:
```diff
@@ -1,169 +1,167 @@
+		if(lstSelectUOM.isDisplayed()){
+			lstSelectUOM.select(strUOMValue);
+			btnAddToCart.jsClick();
+			driver.setPageTimeout(3,TimeUnit.SECONDS);
```
Remove the setPageTimeout line please

jessicamarshall commented on src/main/java/com/xeeva/catalog/ItemDetailsPage.java:
```diff
@@ -1,169 +1,167 @@
+	 */
+	public void addPriceAgreementItemToCart_And_Verify(){
+		btnAddToCart.jsClick();
+		driver.setPageTimeout(3,TimeUnit.SECONDS);
```
Remove the page timeout line please

jessicamarshall commented on src/main/java/com/xeeva/catalog/ItemDetailsPage.java:
```diff
@@ -1,169 +1,167 @@
+
+	public void  modify_ItemDetails(String UP,String Qty,String UOM){
+		txtunitPrice.clear();
+		txtunitPrice.isDisplayed();
```
If you are not doing anything with the return value of isDisplayed(), then there is no need to use this method.  please remove.

jessicamarshall commented on src/main/java/com/xeeva/catalog/ItemDetailsPage.java:
```diff
@@ -1,169 +1,167 @@
+		modify_ItemDetails(UnitPrice,Quantity,strUOMValue);
+		btnAddToCart.isDisplayed();
+		btnAddToCart.click();
+		driver.setPageTimeout(5);
```
Remove this setPageTimeout line please

jessicamarshall commented on src/main/java/com/xeeva/catalog/RequisitioningPage.java:
```diff
@@ -479,7 +476,7 @@ public void cancelRequisitionFromRecentOrders(String comments){
+					driver.setElementTimeout(2);
 					driver.findElement(By.xpath("//*[@id='gvRecentOdersGrid']/tbody"
 							+ "/tr["+ row +"]/td[13]/div/a/i")).jsClick();
 					//Handle Alert if present
```
Please set the element timeout back to the default - driver.setElementTimeout(Constants.ELEMENT_TIMEOUT

jessicamarshall commented on src/main/java/com/xeeva/catalog/SearchItems/GlobalItemsTab.java:
```diff
@@ -1,131 +1,135 @@
+	public void click_AddToCartButton(){
+		AddToCart.syncVisible(10, false);
+		AddToCart.click();
+		driver.setPageTimeout(3);
```
Remove this setPageTimeout line please

jessicamarshall commented on src/main/java/com/xeeva/navigation/MainNav.java:
```diff
@@ -1,309 +1,316 @@
+				TestReporter.logStep( "Item Number Fronm Cart Rows : "+links.get(0).getText());
+				if(links.get(0).getText().equalsIgnoreCase(itemNumber)){
+					// clicking on Cart Item Edit
+					driver.setPageTimeout(3);
```
Remove this setPageTimeout line please.  You can use the setElementTimeout() method instead, but set it back to the default afterwards

PraveenNamburi committed:
Removed all the driver.setPageTimeouts(); statements in CostCenterPage
and RequisitioningPage classes.

PraveenNamburi committed:
1.Added a new method in RequisitioningPage - copyRFQCancelledItem.
2.Added a new method in CostCenterPage - verifyEDITlinkEnabledOrNot.
3.Deceloped a new test - 'Verify_EditRequisition_LineLevel' in catalog
module.

LalithaBanda committed:
Added new methods

LalithaBanda committed:
Developed new Test "Verify_AddMoreItems", Updated WorkBook
"UpdateCostCenter"

LalithaBanda committed:
Pushing line Endings

PraveenNamburi committed:
Removed line spaces in RecentOrderInformationPage.

LalithaBanda committed:
Merge branch 'lalitha' of https://github.com/Orasi/Xeeva.git into
lalitha

Conflicts:
	src/main/java/com/xeeva/catalog/CostCenterPage.java
	src/main/java/com/xeeva/catalog/RecentOrderInformationPage.java
	src/main/java/com/xeeva/catalog/RequisitioningPage.java
	src/test/java/catalog/cartCheckoutProcess/Verify_SaveCart.java

LalithaBanda committed:
pdated with sync timeout

PraveenNamburi committed:
1.Developed a new test in catalog module -
VerifyReq_CopyItemsForRFQCancelledStatus.
2. Added a new method in CartInformationPage -
verifyAddedCartItem_ForRejectedOrder.
3. Added a new method in ItemDetailsPage -
addCopiedItemToCart_RejectedOrders.
4. Added a new method in RequisitioningPage - copyRFQCancelledItem.

PraveenNamburi committed:
Merge branch 'lalitha' of https://github.com/Orasi/Xeeva.git into
lalitha

Conflicts:
	src/main/java/com/xeeva/catalog/ItemDetailsPage.java
	src/main/java/com/xeeva/catalog/RecentOrderInformationPage.java
	src/main/java/com/xeeva/catalog/RequisitioningPage.java

LalithaBanda committed:
Test InProgress - "Check_SplitOrder_AsPerItemType"

LalithaBanda committed:
Merge branch 'lalitha' of https://github.com/Orasi/Xeeva.git into
lalitha

Conflicts:
	src/main/java/com/xeeva/catalog/CostCenterPage.java

LalithaBanda committed:
pushing unsaved changes

LalithaBanda committed:
Merge branch 'lalitha' of https://github.com/Orasi/Xeeva into stage

# Conflicts:
#	src/main/java/com/xeeva/catalog/CartInformationPage.java
#	src/main/java/com/xeeva/catalog/RecentOrderInformationPage.java
#	src/main/java/com/xeeva/catalog/RequisitioningPage.java
#	src/main/java/com/xeeva/catalog/SearchItems/LocalItemsTab.java
#	src/test/java/catalog/cartCheckoutProcess/ChangeQuantity_ExistingLineLevelRequisition.java
#	src/test/java/catalog/cartCheckoutProcess/Verify_CopiedItem_LineLevel.java
#	src/test/java/catalog/cartCheckoutProcess/Verify_ItemDelete.java

jessicamarshall commented:
Can you guys remove all the setPageTimeout() methods from the https://github.com/Orasi/Xeeva/blob/stage/src/main/java/com/xeeva/catalog/ItemDetailsPage.java page?

LalithaBanda committed:
Developed new test "Check_spltOrder_AsPerItemType"

PraveenNamburi committed:
1.Developed the following tests in catalog module:
  1.VerifyReq_UpdateQuantityAndSaveCart.
  2.CopyItem_RecentOrdersList.
   
2.Added the following methods in RequisitionPage:
  a.getQtyNumber.
  b.re_enterQuantity.

3.Added a new method in CostcenterPage - verifyUpdatedQuantity(). 

4.Added a new method in RecentOrderInformation page -
click_CopyItemLink().

5.Added a new method in CartInformationPage -
updateQuantityAndVerifySaveCartFunc().

PraveenNamburi committed:
Added a new method in CartInformationPage -
updateQuantityAndVerifySaveCartFunc().

PraveenNamburi committed:
Merge branch 'lalitha' of https://github.com/Orasi/Xeeva.git into lalitha

PraveenNamburi committed:
Updated the following workbook with new test data -
VerifyReq_UpdateQuantityAndSaveCart.

PraveenNamburi committed:
1. Developed a new test in catalog module -
VerifyReq_UpdateQuantityAndSaveCart().
2. Added a new method in CartInformationPage - verify_UpdatedQuantity().

PraveenNamburi committed:
Added file to index.

PraveenNamburi committed:
Moved the following tests into RecentOrdersTab package:
1.CancelRequisition_RecentOrdersList.
2.CopyItem_RecentOrdersList.

PraveenNamburi committed:
1.Updated the test case - VerifyReq_UpdateQuantityAndSaveCart.
2.Added the following new methods in CartInformationPage:
   a.deleteExistingCartItems(), and
   b.verify_DeleteCartItems().

LalithaBanda committed:
Updated Comments, Added new methods

LalithaBanda committed:
Developed new Test "Check_SplitOrder_AsPerItemType"

LalithaBanda committed:
Developed new Test "Verify_CheckOutDetails_AddMoreItems"

LalithaBanda committed:
pushing unsaved changes

LalithaBanda committed:
Merge branch 'lalitha' of https://github.com/Orasi/Xeeva into stage

# Conflicts:
#	src/main/java/com/xeeva/catalog/CostCenterPage.java
#	src/main/java/com/xeeva/catalog/RecentOrderInformationPage.java
#	src/test/resources/catalog/cartCheckoutConfirmRequest/Check_SplitOrder_AsPerItemType.xml
